test/op_shape_test.cpp

```diff
@@ -2525,6 +2525,15 @@ TEST_CASE(qlinear_fp16)
     expect_shape(result, migraphx::make_op("quantizelinear"), input, scales);
 }
 
+TEST_CASE(qlinear_output_shape)
+{
+    migraphx::shape scales{migraphx::shape::half_type, {2, 4}};
+    migraphx::shape input{migraphx::shape::half_type, {2, 4}};
+    migraphx::shape result{migraphx::shape::int8_type, {2, 4}};
+    expect_shape(
+        result, migraphx::make_op("quantizelinear", {{"out_type", result.type()}}), input, scales);
+}
+
```

Comment:
minor: could add a test that sets the `out_type` attribute.

Reply:
Done.